js/lib/jupyter-aladin.js

```diff
@@ -30,7 +30,7 @@ const loadScript = (FILE_URL, async = true, type = "text/javascript") => {
         }
     });
 };
-var AladinLiteJS_Loader = loadScript("https://code.jquery.com/jquery-3.6.1.min.js")
+let AladinLiteJS_Loader = loadScript("https://code.jquery.com/jquery-3.6.1.min.js")
```

Comment:
Just a comment that we do not need to load jQuery anymore as it is bundled in the aladin.js loaded afterwards. I think that should be done in a separate commit.

Reply:
Yup, you want to push that to the dev branch ? Or do I do it ?

Reply:
I can remove that in a separate commit